pypdns/__init__.py,sha256=PzDlH1ny616XH6_fK17VEiHuvMTBM_JWI3ZpOe3sHbY,134
pypdns/api.py,sha256=UruVKNriu94PQ026ODpFxjW6qm1U7f0CdTmS6qw8Haw,11776
pypdns/errors.py,sha256=z1MDSVuN0I553AczIVQhJA4U7JPoChkRAv1dOFaNLPw,474
pypdns/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pypdns-2.1.0.dist-info/LICENSE,sha256=o1p6jKkqoo0k2KXZ23F8zTPaRGIoF8Gu9ADxi-QiUyo,1576
pypdns-2.1.0.dist-info/METADATA,sha256=8bAFGGzee_HSQQt9DWSu08pDGq3GyRzZrQ1kCtx1jYQ,1631
pypdns-2.1.0.dist-info/WHEEL,sha256=Zb28QaM1gQi8f4VCBhsUklF61CTlNYfs9YAZn-TOGFk,88
pypdns-2.1.0.dist-info/RECORD,,
